src/components/menu/__internal__/submenu/submenu.component.tsx

```diff
@@ -200,18 +210,36 @@ const Submenu = React.forwardRef<
 
         // Get the last menu item in the last segment block
         const lastMenuItemInSegmentBlock = segmentBlockMenuItems.pop();
+
+        // Check to see if the last menu item in the last segment block is visible
+        let isLastMenuItemInSegmentBlockVisible = false;
+        if (lastMenuItemInSegmentBlock && lastSegmentBlock) {
+          isLastMenuItemInSegmentBlockVisible =
+            lastMenuItemInSegmentBlock.getBoundingClientRect().bottom <
+            lastSegmentBlock.getBoundingClientRect().bottom;
+        }
+
         // Check if the last item in the segment block is the same as the last MenuItem in the submenu
         const menuItemsMatch =
           !!lastMenuItemInSegmentBlock &&
           lastMenuItemInSegmentBlock === lastMenuItem;
 
+        // Applies the focus radius to the last segment block
```

Comment:
**suggestion(non-blocking):** This comment needs changing as the `applyFocusRadius` is being used in the `StyledBox` of the `StyledScrollableBlock`. I'd say it should be changed to reflect that.

Reply:
done.